python/tests/test.py

```diff
@@ -592,12 +593,20 @@ def test_basic(self):
         self.assertEqual(self.config.output.spikes_file, 'out.h5')
 
         self.assertEqual(self.config.report('soma').cells, 'Mosaic')
-        self.assertEqual(self.config.report('soma').type, 'compartment')
+        self.assertEqual(self.config.report('soma').type, Report.Type.compartment)
+        self.assertEqual(self.config.report('soma').compartments, Report.Compartments.center)
```

Comment:
I am wondering how we are going to use the enum values in neurodamus?
Use the enum type directly
`self.assertEqual(self.config.report('soma').type, Report.Type.compartment)`
or still convert to string?
`self.assertEqual(self.config.report('soma').type.name, 'compartment')`

Reply:
I created a neurodamus-py MR for that
https://bbpgitlab.epfl.ch/hpc/sim/neurodamus-py/-/merge_requests/47

Reply:
Good, I see.  Can you also give an example in the python test here using `.name` ?